applications/location-api/README.md

```diff
@@ -0,0 +1,161 @@
+
+If the installation succeds, test it either opening a browser to `<ip-address>:<edge-node-port>` for seeing the GUI or typing in a terminal the command `$ curl <ip-address>:<edge-node-port>`.  
+
+### Start a simulation
+The GUI allows to start a simulation. Going to `<ip-address>:<edge-node-port>`, you will see the GUI, which should look like the following: 
+
+![Alt text](./simulator_gui.PNG)
+
+The GUI allows to create and simulate the movement of mobile devices through a given path or track. The track must be a well-formed GPX file, and it must include at least *coordinates* and the relative *timestamp*, like the following: 
+
+```xml
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<gpx version="1.0" creator="GPS Visualizer https://www.gpsvisualizer.com/" xmlns="http://www.topografix.com/GPX/1/0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.topografix.com/GPX/1/0 http://www.topografix.com/GPX/1/0/gpx.xsd">
+	<trk>
+  		<name>sample</name>
+  		<trkseg>
+			<trkpt lat="45.072883333" lon="7.78415">
+				<time>2013-01-30T15:41:38.9Z</time>
+			</trkpt>
+			<trkpt lat="45.0729" lon="7.784133333">
+				<time>2013-01-30T15:41:39Z</time>
+			</trkpt>
+			<!-- other track points here -->
+		</trkseg>
+	</trk>
+</gpx>
+```
+
+> In the beginning, the simulator contains just one GPX file as a sample, but you can add as many GPX files as you want, going to the *Upload file* section. In the upper right, from the menu, you can go there and upload it. Then, it will be available in the *home* section. 
+
+For starting a simulation you need to: 
+
+- write down a name for each *`Station ID`* and select a GPX file from the dropdown list
+- add as many *`Station ID`* as you want to simulate
+- choose if make the simulation iteratively *`Make it as loop`* or run just once
+- click on button *`Start`*
+
+The simulator will start feeding location information to the Location API service.
+
+### Install consumer sample application 
+The consumer sample application can be easily deployed through the provided YAML file. The YAML file is the simplest one, but it has a *mandatory*  field that enables the communication from *Location API Simulator Pod* and *Location API consumer Pod*. Below, the YAML file is shown: 
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+    name: consumer
+    labels:
+  		locationService: active
+spec:
+    containers:
+    -
+        image: "location-api-consumer:latest"
+        imagePullPolicy: Never
+        name: consumer
+            ports:
+                - containerPort: 8082
+```
+
+Therefore, the label `locationService: active` **must be included** for using the Location API Simulator. All the other labels can be changed according to the proper needs.
+
+From the *Controller Node*, execute the following command: 
+
+```sh
+$ kubectl create -f consumer.yaml
```

Comment:
It might be easier to include the consumer app too in the Helm for quick evaluation. But, you could have better reasons to keep them separate - up to you.

Reply:
Yes, I prefer to keep separeted the two files, since it is just an example. If It is ok, I leave there it is